web/genericcom.js

```diff
@@ -25,7 +25,10 @@ if (typeof PDFJSDev !== "undefined" && !PDFJSDev.test("GENERIC")) {
   );
 }
 
-const GenericCom = {};
+// let viewerApp = { initialized: false };
```

Comment:
Can't we simply remove the comments and reduce this to `function initCom(app) {}` (similar to the interface file which also has a no-op implementation)?

Reply:
Sure, it can be removed; I just left that to document things a little bit.